README.md

```diff
@@ -129,6 +129,17 @@ You can find a further CRUD example in the [example](https://github.com/cloudant
 
 ### Initialization
 
+#### 1. Using a URL:
+
+You can also initialize Cloudant with a URL:
+
+~~~ js
+var Cloudant = require('cloudant')
+var cloudant = Cloudant("http://MYUSERNAME:MYPASSWORD@localhost:5984");
+~~~
+
+#### 2. Using Cloudant account credentials:
+
 To use Cloudant, `require('cloudant')` in your code. That will return the initialization function. Run that function, passing your account name and password, and an optional callback. (And see the [security note](#security-note) about placing your password into your source code.
```

Comment:
Think this bit needs to move before the numbers:
> To use Cloudant, `require('cloudant')` in your code. That will return the initialization function. Run that function

and maybe add "with one of the following options" and then the "and an optional callback".